src/Packetery/Module/Order/Form.php

```diff
@@ -134,4 +135,32 @@ public function setDefaults(
 			]
 		);
 	}
+
+	/**
+	 * Gets invalid fields from validation result.
+	 *
+	 * @param array<string, string> $validationResult Validation result.
+	 *
+	 * @return array
+	 */
+	public static function getInvalidFieldsFromValidationResult( array $validationResult ): array {
+		$validationFormInputMapping = [
+			self::FIELD_VALUE  => Order::ERROR_TRANSLATION_KEY_VALUE,
+			self::FIELD_WEIGHT => Order::ERROR_TRANSLATION_KEY_WEIGHT,
+			self::FIELD_HEIGHT => Order::ERROR_TRANSLATION_KEY_HEIGHT,
+			self::FIELD_WIDTH  => Order::ERROR_TRANSLATION_KEY_WIDTH,
+			self::FIELD_LENGTH => Order::ERROR_TRANSLATION_KEY_LENGTH,
+		];
+
+		$fields = [];
+		foreach ( $validationFormInputMapping as $fieldName => $value ) {
+			if ( ! isset( $validationResult[ $value ] ) ) {
+				continue;
+			}
+
+			$fields[ $fieldName ] = $fieldName;
```

Comment:
Nelíbí se mi moc klíč stejný jako hodnota, ale asi můžeme nechat, zaplánujeme vylepšení těch hlášek a upraví se to pak.